.github/workflows/main.yml

```diff
@@ -336,7 +336,6 @@ jobs:
         run: |
           cargo test --manifest-path="wallet_migrator/Cargo.toml";
           RUST_TEST_THREADS=1 CARGO_INCREMENTAL=0 TEST_POOL_IP=127.0.0.1 cargo test --manifest-path="aries_vcx/Cargo.toml" -F migration --test test_creds_proofs -- --include-ignored;
-          RUST_TEST_THREADS=1 CARGO_INCREMENTAL=0 TEST_POOL_IP=127.0.0.1 cargo test --manifest-path="aries_vcx/Cargo.toml" -F migration --test test_creds_proofs_revocations -- --include-ignored;
```

Comment:
how come we're removing this? is it now failing?

Reply:
good catch, reverted. 

I mistakenly thought it's working on prefix basis, thinking I've removed duplicated test execution, while obviously it's specifying specific test files.